frontend/src/components/common/NameValueTable/NameValueTable.tsx

```diff
@@ -123,7 +65,10 @@ export default function NameValueTable(props: NameValueTableProps) {
     <Grid
       container
       component="dl" // mount a Definition List
-      className={classes.table}
+      sx={theme => ({
+        border: '1px solid #e7e7e7',
+        borderRadius: theme.shape.borderRadius,
```

Comment:
Not sure why but this doesn't seem to be working:
![Screenshot showing larger round corners](https://github.com/headlamp-k8s/headlamp/assets/1029635/1d076872-de9a-4f19-a2ed-1b6591e356ce)

Reply:
What's not working with this exactly?

![image](https://github.com/headlamp-k8s/headlamp/assets/9541/b4c1e604-2b50-46e1-9551-796a9f8c601f)

Reply:
I see rounded corners and when disabled `borderRadius: theme.shape.borderRadius` I see normal edged corners

![image](https://github.com/headlamp-k8s/headlamp/assets/78232183/4a648ac5-73f1-40a6-992f-ef6745241f68)